### ISSUE TITLE
read values

### ISSUE THREAD

**Issue by AshishGusain17** [leimao/Frozen_Graph_TensorFlow#9]
Hey, Can you tell how to read the values like weights and gradients using this frozen graph.
We also have other model.ckpt.data   .meta    .index   files.
Is there any way?

**Comment by leimao** [comment]
You could figure out a way to load checkpoint models as a Keras model, then you could try to export a frozen model using the methods I described.

**Comment by AshishGusain17** [comment]
Is there even a way to load inference models as keras models. If so, can you share some articles or docs file link

**Comment by leimao** [comment]
It is not possible to load the frozen model to Keras as far as I know.

**Comment by AshishGusain17** [comment]
So, how can I load weights to keras model.
I have tensorflow model available i.e. model.ckpt,data,frozen graph,checkpoint.
If not load, how can I get the values of weights and gradients in each layer

**Comment by felipetobars** [comment]
HI @leimao, thanks for the documentation, I wanted to ask you if there is any way to make batch predictions using the freeze model, similar to how the predict() function is used with the 'batch_size=' parameter
